code-reviews/example1/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>All About Broccoli</title>
+    <link rel='stylesheet' type='text/css' href='styles.css'>
+</head>
+<body class='body'>
+    <div class='main'>
+        <h2>Broccoli 🥦</h2>
```

Comment:
We could even add something like, 'Nothing Beats A Well Made Broccoli! ' With as smiley face emoji.